### PR TITLE
GC/MS-documentation update 191114

### PR DESCRIPTION
**Pull request by MelissaESPOSITO**

### CODE REVIEW

File: docs/gc-ms-library-molecular-network.md [hunk 1]
```diff
@@ -11,13 +11,13 @@ There are two options to open the job page for conducting a spectral library sea
...
 
 (3) Select your metadata table formatted as a tab-separated .txt file. To create your metadata table properly, see the documentation [here](metadata.md). 
 
-(4) Library files have to be added. Each user should add their libraries on GNPS by following the instructions [here](batchupload.md), or by uploading them in .mgf format (drag and drop on “Upload File” section). 
+(4) Public Libraries files are selected by default. However, user could add their libraries on GNPS by following the instructions [here](batchupload.md), or by uploading them in .mgf format (drag and drop on “Upload File” section). 
 
-(5) Optional carbon marker table. Format described below.
+(5) The carbon marker table should be added. This table must be formatted as a comma separated values (.csv) file in the following headers: Compound_Name - name of the compound (string), Carbon_Number - number of carbons (integer), RT - retention time in seconds (float). [Example File](static/gc_kovats_skin.csv). The file can also be created from the raw experimental chromatograms of the RI marker compounds using MZMine2 as described in this [video tutorial](https://youtu.be/XodHMJcuwnk).
```
Comment (mwang87) [inline docs/gc-ms-library-molecular-network.md]: Can you check that the video tutorial is consistent with the format?

File: docs/gc-ms-library-molecular-network.md [hunk 2]
```diff
@@ -81,7 +81,7 @@ The **job window** proposed four fields of advanced search, filtering, network,
 
 ## Inspect the Results
 
-After the job has been completed, the **job status page** gives access to three different views:
+After the job has been completed, the **job status page** gives access to four molecular networking results views:
```
Comment (mwang87) [inline docs/gc-ms-library-molecular-network.md]: say several views

File: docs/gc-ms-library-molecular-network.md (hunk 1 above)
Comment (MelissaESPOSITO) [reply docs/gc-ms-library-molecular-network.md]: I'll check

Comment (MelissaESPOSITO) [reply docs/gc-ms-library-molecular-network.md]: He said nothing about the format of the table so that ok!

**Pull request merged**